products/silverstripe.md

```diff
@@ -74,6 +74,41 @@ releases:
     latest: "4.10.11"
     latestReleaseDate: 2022-11-21
 
+-   releaseCycle: "3.7"
+    releaseDate: 2018-06-08
+    support: true
+    eol: 2021-09-30
```

Comment:
Where can I confirm this date ?

Reply:
http://web.archive.org/web/20200930101626/https://www.silverstripe.org/software/roadmap/